src/articles/subjects/science/chemistry/topic-1/2-elements-less-complicated-than-they-sound.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Elements: they're way less complicated than they sound"
+date: "2019-11-10"
+description: "An element is a substance made up of atoms that all have the same number of protons in their nucleus."
+subject: "chemistry"
+subjectGroup: "science"
+topic: "topic 1 - atomic structure and the periodic table"
+---
+
+# What is an element?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4474002)